TLM/TLM/UI/DebugSwitches/DebugSwitchCheckboxOption.cs

```diff
@@ -0,0 +1,26 @@
+#if DEBUG
+namespace TrafficManager.UI.DebugSwitches {
+    using ColossalFramework;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using TrafficManager.State;
+    using TrafficManager.State.ConfigData;
+    using TrafficManager.UI.Helpers;
+
+    internal class DebugSwitchCheckboxOption : CheckboxOption {
+        internal DebugSwitch DebugSwitch;
+        public DebugSwitchCheckboxOption(DebugSwitch sw)
+            : base(sw.ToString(), Options.PersistTo.Global) {
```

Comment:
Hmm, I'm pretty sure you've said it won't be saved into config so PersistTo.Global is wrong. It requires implementation of dedicated save action anyways, which I don't see but still, IMO it should be None if we are not saving it in the config

Reply:
@kianzarrin Ok, just what I thought, you need to add handler for setting the actual debug switch in the config. Currently I can only click checkboxes but don't see any effect because the switch state change is not propagated to `DebugSettings.DebugSwitch` flag 😃 
Other than that major issue, everything else seems to be working (modals etc.).

Reply:
as I have suggested in the later PR, `PersistTo` should be renamed to Scope because that is what it really means.

Reply:
so this is not working?
```cs
Handler = val => DebugSwitch.SetFlags(DebugSwitch, val);
```

Reply:
oh damn is changing something else

Reply:
@krzychu124 now is working :)
```
Debug 598.1356474: DebugSettings.DebugSwitches set to CalculateSegmentPosition
```